packages/project-management-automation/lib/tasks/first-time-contributor-label/index.js

```diff
@@ -46,6 +46,22 @@ async function firstTimeContributorLabel( payload, octokit ) {
 		issue_number: payload.pull_request.number,
 		labels: [ 'First-time Contributor' ],
 	} );
+	
+/**
+ * Adds a welcome comment to the first time PR
+ */
+	
+	await octokit.issues.createComment( {
+		owner,
+		repo,
+		issue_number: payload.pull_request.number,
+		body: ':wave: Thanks for your first Pull Request and for helping build the future of Gutenberg and WordPress, @' +
+		author +
+		"!. In case you missed it, we'd love to have you join us in our [Slack community](https://make.wordpress.org/chat/), +
```

Comment:
Just missing a quotation mark at the end of this line.

```suggestion
		"!. In case you missed it, we'd love to have you join us in our [Slack community](https://make.wordpress.org/chat/)," +
```

Reply:
You'll have to fix that before following @noisysocks' suggestion, as the formatting tool will only work on valid code.

Once that's done, hopefully should be good to go 🎉